commands/support_bundle.go

```diff
@@ -51,11 +63,27 @@ func getFlags() []components.Flag {
 			Name:        retryInterval,
 			Description: "The duration to wait between retries",
 		},
+		components.BoolFlag{
+			Name:         cleanup,
+			Description:  "Delete generated support bundle",
```

Comment:
_Delete the support bundle local temp file after upload._ or something like that.

Reply:
fix